### PR TITLE
Expose api-versions and api-resources in oc

### PR DESCRIPTION
**Pull request by soltysh**
/assign @deads2k

### CODE REVIEW

File: pkg/oc/cli/cmd/wrappers.go [hunk 1]
```diff
@@ -788,3 +789,39 @@ func NewCmdAuth(fullName string, f *clientcmd.Factory, out, errout io.Writer) *c
...
+		# Print the supported non-namespaced resources
+		%[1]s api-resources --namespaced=false
+
+		# Print the supported API Resources with specific APIGroup
+		%[1]s api-resources --api-group=extensions`)
+)
+
+func NewCmdApiVersions(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
+	cmd := kcmd.NewCmdApiVersions(f, out)
+	cmd.Example = fmt.Sprintf(apiresourcesExample, fullName)
```
Comment (deads2k) [inline pkg/oc/cli/cmd/wrappers.go]: why bother replacing the example?

**Comment by deads2k** [comment]
is this api-resources in it's current state or the state after the first pull?  I suspect we'll want the current state.  You could do it as two drops on top of this:  one drop for copy/paste of current state.  a second drop for fitting to the old factory if any.

File: pkg/oc/cli/cmd/wrappers.go (hunk 1 above)
Comment (soltysh) [reply pkg/oc/cli/cmd/wrappers.go]: Because the kube version uses kubectl in examples.

[the remark this answers is not in the document]
Comment (soltysh) [reply pkg/oc/cli/cmd/wrappers.go]: My bad.

**Comment by soltysh** [comment]
@deads2k the picks were very easy so I just went with UPSTREAM commits.

**Comment by deads2k** [comment]
@soltysh @mfojtik we should have a card to make a better way to handle the binary name substitution in kubectl help.  Perhaps we can simply write a dumb find/replace on the command tree.

You should have a test-cmd test that demonstrates this working on openshift resources.

lgtm otherwise.  feel free to tag after test.

**Comment by soltysh** [comment]
Test added, tagging.

**Pull request merged**